geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/CommandManager.java

```diff
@@ -128,15 +134,17 @@ private static void raiseExceptionIfEmpty(Set<?> foundClasses, String errorFor)
     return userCommandPackages;
   }
 
-  private void loadUserCommands(ClasspathScanLoadHelper scanner, Set<String> restrictedToPackages) {
-    if (restrictedToPackages.size() == 0) {
+  private void loadUserDefinedCommands() {
+    String[] userCommandPackages = getUserCommandPackages().toArray(new String[] {});
+
+    if (userCommandPackages.length == 0) {
       return;
     }
 
     // Load commands found in all of the packages
-    try {
+    try (ClasspathScanLoadHelper scanner = new ClasspathScanLoadHelper(userCommandPackages)) {
       Set<Class<?>> foundClasses = scanner.scanPackagesForClassesImplementing(CommandMarker.class,
-          restrictedToPackages.toArray(new String[] {}));
+          userCommandPackages);
```

Comment:
I believe you don't need to pass in the 2nd parameter if scanning from all packages